src/hooks/screen/useChatScreenHook.ts

```diff
@@ -35,8 +35,13 @@ export const ScrollContext = React.createContext<ScrollContextProps | null>(null
 
 function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
   const {localDb, refresh, otherListener} = useLocalDatabaseHook();
-  const {selectedChannel, goBackFromChatScreen, goToChatInfoScreen, setChannelAsRead} =
-    useChatUtilsHook(type);
+  const {
+    selectedChannel,
+    goBackFromChatScreen,
+    goToChatInfoScreen,
+    setChannelAsRead,
+    setSelectedChannel
+  } = useChatUtilsHook(type);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `setSelectedChannel` from `useChatUtilsHook(type)` seems to be a good change. It's used later in the cleanup function of a `useEffect` hook, which is a common pattern for resetting state when a component unmounts.

```typescript
const {
  selectedChannel,
  goBackFromChatScreen,
  goToChatInfoScreen,
  setChannelAsRead,
  setSelectedChannel
} = useChatUtilsHook(type);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->